dev/io.openliberty.jaxws.security_fat.ssl/publish/files/JaxWsTransportSecurityServer/bindings/certInClientKSButNotInServerTS.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same comment about verifying that all these binding files are used in this test project, and if any aren't delete the ones not used to avoid duplicates.